packages/css/src/plugins/atomicify-rules.ts

```diff
@@ -216,7 +216,7 @@ export const atomicifyRules = (opts = {}): Plugin => {
       root.each((node) => {
         switch (node.type) {
           case 'atrule':
-            const supported = ['media', 'supports', 'document'];
+            const supported = ['media', 'supports', 'document', '-moz-document'];
```

Comment:
I was hoping that we can support all atrules instead of keeping adding them to the list in the future. At least, if we don't support a certain atrule, we should warn the developers (or even throw an error) rather than just not working silently (I think it's what it does currently).

Hey @itsdouges , it's probably a big throwback. From your [commit](https://github.com/atlassian-labs/compiled/pull/299/files#diff-55e7dd703c9c2042f0a2ba361fc8d6e322fe3de404cc2962ea005b424b4b3ab1R207), do you remember why we have a limited support for atrules? Can we remove `if (supported.includes(node.name))`?

Reply:
oh, actually I think i understand why now. I guess it's because some atrules don't make sense to be atomicified.

e.g. `@import url;`

Reply:
> oh, actually I think i understand why now. I guess it's because some atrules don't make sense to be atomicified.
Yeahhh, I was just looking at [At-rules](https://developer.mozilla.org/en-US/docs/Web/CSS/At-rule); I think you are spot on.

Wondering if there is any other case that we are missing

Reply:
> At least, if we don't support a certain atrule, we should warn the developers (or even throw an error) rather than just not working silently (I think it's what it does currently).

Ah that's a good point @liamqma - I'll revert this PR to a draft and see if I can add some error messaging for unsupported at rules

Reply:
Out of curiosity, would it make sense for [`@container`](https://developer.mozilla.org/en-US/docs/Web/CSS/@container) and [`@layer`](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer), to be atomificied?

Looks like they support syntax like this:
```
@layer state {
    .warning {
        background-color: brown;
    }
    p {
        text-align: right;
    }
}
```

Reply:
Yes, I think so. Both makes sense to be atomificied.
In fact, devs from Atlassian have asked for it. I'll dm you the message.